server/routes/testutils/appSetup.ts

```diff
@@ -16,6 +16,7 @@ import setApis from '../../middleware/setApis'
 
 export const testAppInfo: ApplicationInfo = {
   applicationName: 'hmpps-incident-reporting',
+  productId: 'test-product-id',
```

Comment:
product id is `DPS021` (not that it really matters for tests)

Reply:
yeah, exactly, this is just for test